android/build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
         }
 
         dependencies {
-            classpath("com.android.tools.build:gradle:3.5.3")
+            classpath("com.android.tools.build:gradle:4.0.1")

```

Comment:
can we leave out the gradle change?  
I'm not sure if there are any implications from this, but it is also not related to the PR